src/sass/components/_container.scss

```diff
@@ -1,9 +1,7 @@
 .container {
-  display: flex;
-  align-items: center;
-  justify-content: center;
   width: 100%;
```

Comment:
Wiesz co znalazłem też taki błąd jak jest te 100% width to te paddingi rozpychają rodzica. Można to łatwo naprawić a będzie działać na tej samej zasadzie. Te zapytanie medialne min width: 480px trzeba całkiem usunąć, a width 100% zmienić na max-width: 480px i wtedy jest git.